### PR TITLE
Registration enhancement

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: app/i18n/en.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "shareYourAddress": "Share your address"
   },
   "RegistrationScreen": {
-    "likerIdLabel": "Create your Liker ID (7 or more characters)"
+    "likerIdHint": "Allow 7-20 lowercase letters, dash and underscore only",
```
Comment (nnkken) [inline app/i18n/en.json]: number are also allowed

**Comment by williamchong007** [comment]
how about also call check username API?
https://documenter.getpostman.com/view/6879252/SVzxZfwH?version=latest#ae71174e-7e84-42bb-b541-6ec74b209175

**Comment by nwingt** [comment]
> https://documenter.getpostman.com/view/6879252/SVzxZfwH?version=latest#ae71174e-7e84-42bb-b541-6ec74b209175

@williamchong007 I think we can put this to next sprint

File: app/i18n/zh-HK.json [hunk 2]
```diff
@@ -42,7 +42,8 @@
   },
   "RegistrationScreen": {
     "likerIdHint": "只允許7至20位小寫英文字母、數字、「-」及「_」字元",
-    "likerIdLabel": "創建 Liker ID"
+    "likerIdLabel": "創建 Liker ID",
+    "shouldSignInOnDesktop": "請在電腦上登入 http://like.co 後再試一遍"
```
Comment (williamchong007) [inline app/i18n/zh-HK.json]: Do we want to mention `email exists` in the message?

Comment (nwingt) [reply app/i18n/zh-HK.json]: This is the body message, and the title is "email exists"

**Pull request merged**